cmd/bosun/sched/check.go

```diff
@@ -185,6 +186,44 @@ func (s *Schedule) RunHistory(r *RunHistory) {
 	if checkNotify && s.nc != nil {
 		s.nc <- true
 	}
+	// Instrument alert states.
```

Comment:
Can we factor all of this into a seperate method? RunHistory is already too big IMO. `s.CollectMetrics()` or something may be preferable to inlining it here.